test/python/WMCore_t/DataStructs_t/File_t.py

```diff
@@ -32,8 +34,8 @@ def testDefinition(self):
         self.assertEqual(testFile['size'], 0)
         self.assertEqual(testFile['events'], 0)
         self.assertEqual(testFile['checksums'], {})
-        self.assertItemsEqual(testFile['parents'], {})
-        self.assertItemsEqual(testFile['locations'], {})
+        self.assertCountEqual(testFile['parents'], {}) if PY3 else self.assertItemsEqual(testFile['parents'], {})
```

Comment:
Please be honest ;) Would you like to update it to the override version (that you provided today in a different PR):
```
if PY3:
    self.assertItemsEqual = self.assertCountEqual
```
? You have already made all the hard changes, so I'm happy to deal with this in the future as well.

Reply:
I do agree with you, I'd better fix this as you say, thanks Alan!

Reply:
(pushed in the last commits, already squashed)